pages/plugins.rst

```diff
@@ -34,14 +49,26 @@ What you need in your development environment before starting is:
 
 There are lots of different ways to get those on your local machine, unfortunately we cannot list all of them, so please refer to your operating system-specific documentation,
 
+Graylog uses a couple of conventions and techniques in its code, so be sure to read about the :ref:`general_concepts_api` for an overview.
+
+.. _sample_plugin:
+
+Sample Plugin
+-------------
+
+To go along with this documentation, there is a `sample plugin on Github <https://github.com/Graylog2/graylog-plugin-sample/tree/2.2>`_. Tis documentation will link to specific parts for your reference.
+It is fully functional, even though it does not implement any real logic. Its purpose so to give a reference for helping to implement your own plugins.
```

Comment:
"it does not implement any real logic" &rarr; "it does not implement any useful functionality"